### DISCUSSION TITLE
Install / Use AppFlowy in Windows

### DISCUSSION THREAD

**Discussion by prishs** [AppFlowy-IO/AppFlowy#1598] [General help wanted]
I have been trying to use AppFlowy app on Windows 10 21H2 since last couple of releases.. But the app just doesn't open in windows.
I tried using .zip & .exe file.. It just flashes blank white screen for 5-10 sec & quits.

Am I missing any part while installing??

**Comment by annieappflowy** [comment]
Thanks for reporting! We're looking into it.
